apps/docs/pages/guides/functions/cicd-workflow.mdx

```diff
@@ -41,6 +43,68 @@ jobs:
       - run: supabase functions deploy --project-ref $PROJECT_ID
 ```
 
+## GitLab CI
+
+Here is the sample pipeline configuration to deploy via GitLab CI.
+
+```yaml
+
+image: node:18
+
+# List of stages for jobs, and their order of execution
+stages:
+  - setup
+  - deploy
+
+# This job runs in the setup stage, which runs first.
+setup-npm:
+  stage: setup
+  script:
+    - npm i supabase@beta
```

Comment:
Should we recommend `beta` in the docs @sweatybridge? kinda defeats the point of a staged rollout of the cli if we recommend it by default

Reply:
good catch. lets drop the beta in docs

```suggestion
    - npm i supabase
```